viera.py

```diff
@@ -116,6 +116,23 @@ def index(self):
     def hold(self):
         self._sendkey('NRC_HOLD-ONOFF')
 
+    def ch_up(self):
+        self._sendkey('NRC_CH_UP-ONOFF')
+    
+    def ch_down(self):
+        self._sendkey('NRC_CH_DOWN-ONOFF') 
+
+    def input(self):
+        self._sendkey('NRC_CHG_INPUT-ONOFF') 
+
+    def last_view(self):
+        self._sendkey('NRC_R_TUNE-ONOFF') 
+
+    def power_off(self):
+        self._sendkey('NRC_POWER-ONOFF')
+
```

Comment:
The convention regarding the amount of whitespace between top level definitions is to put two blank new lines (not 3)
(see https://www.python.org/dev/peps/pep-0008/#blank-lines)